api/src/main/java/com/github/eyefloaters/console/api/model/Node.java

```diff
@@ -4,10 +4,47 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
 @JsonInclude(value = Include.NON_NULL)
-public record Node(int id, String host, int port, String rack) {
```

Comment:
This changed from a record because the log field is mutable. At this point I'm questioning whether we even need to model a node like this or should it really just be the replica information with only `nodeId`. That's the spirit of what I'm attempting to do here and I think the `Node` concept is misleading. Do we ever care about the host/port/rack information?

Reply:
host and port probably not. Rack could be useful. For example, I could imagine we want to highlight when two replicas are in the same rack, because then the redundancy is less good than it might appear.

Reply:
But, of course, rack being useful doesn't mean we need a `Node` class to expose it.

Reply:
As noted elsewhere, reverted `Node` and added `PartitionReplica` with `nodeId` and `nodeRack`, using the record style naming :-)